bot/exts/moderation/stream.py

```diff
@@ -85,8 +85,9 @@ async def stream(self, ctx: commands.Context, member: discord.Member, duration:
         log.trace(f"Attempting to give temporary streaming permission to {member} ({member.id}).")
         # If duration is none then calculate default duration
         if duration is None:
-            now = datetime.datetime.utcnow()
-            duration = now + datetime.timedelta(minutes=VideoPermission.default_permission_duration)
+            duration = datetime.datetime.utcnow() + datetime.timedelta(
+                minutes=VideoPermission.default_permission_duration
+            )
```

Comment:
This is worse because it's 2 lines versus 3 and putting the argument on a separate line is less readable. Just a comment; not going make you go back and forth on this.